apps/frontend/components/Filters/DisasterFilter.tsx

```diff
@@ -26,56 +26,43 @@ export const DisasterFilter = ({ value, onChange }: Props): JSX.Element => {
   );
 
   return (
-    <RadioGroup
-      value={value}
-      onChange={onChange}
-      sx={{
-        display: 'flex',
-        flexDirection: 'row',
-        justifyContent: 'center',
-        marginBottom: 3,
-      }}
-    >
-      <Card variant="outlined" sx={{ marginRight: 5 }}>
-        <CardContent>
-          <Typography variant="h6" sx={{ marginBottom: 1 }}>
-            <FormattedMessage
-              id="validation_search_params.disaster_type"
-              defaultMessage="Disaster Type"
-            />
-          </Typography>
-          {disasters.map(type => (
-            <FormControlLabel
-              key={type}
-              value={type}
-              control={<Radio />}
-              label={intl.formatMessage({
-                id: `disasters.${type}`,
-              })}
-            />
-          ))}
-        </CardContent>
-      </Card>
-      <Card variant="outlined" sx={{ maxWidth: 600, marginLeft: 5 }}>
-        <CardContent>
-          <Typography variant="h6" sx={{ marginBottom: 1 }}>
-            <FormattedMessage
-              id="validation_search_params.incident_type"
-              defaultMessage="Incident Type"
-            />
-          </Typography>
-          {incidents.map(type => (
-            <FormControlLabel
-              key={type}
-              value={type}
-              control={<Radio />}
-              label={intl.formatMessage({
-                id: `disasters.${type}`,
-              })}
-            />
-          ))}
-        </CardContent>
-      </Card>
-    </RadioGroup>
+    <Box display="flex" flexDirection="row" justifyContent="center" margin={2}>
+      <FormControl>
+        <InputLabel>
+          <FormattedMessage id="validation_search_params.disaster_type" />
+        </InputLabel>
+        <Select
+          value={value}
+          onChange={onChange}
+          sx={{ minWidth: 200 }}
+          label={intl.formatMessage({
+            id: 'validation_search_params.disaster_type',
+          })}
+        >
+          <ListSubheader>
```

Comment:
flood, drought, incident should be at the same level. We could create sub header for flood and drought separately if needed?